test/container_app_test.py

```diff
@@ -19,6 +20,11 @@
 def my_f_1(x):
     pass
 
+# Simulate calling `breakpoint()` in test
+def test_breakpoint():
+    return True
+
+sys.breakpointhook = test_breakpoint
```

Comment:
This works but can you use a context manager with mocking instead? This replacement is flaky and may be overwritten by other programs. Example:

```python
with mock.patch("sys.breakpointhook", test_breakpoint):
   # your test code here
```